### PR TITLE
Card object and array iteration scripts to populate index.html file

### PR DESCRIPTION
**Pull request by paulfleurdelys**
Lesson 03 assignment

### CODE REVIEW

File: lesson03/assignment/index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+const localData = [
```
Comment (tylervanbrocklin) [inline lesson03/assignment/index.js:1]: You seem to have the data in two places, making your code slightly redundant. Both in your data.js file and hardcoded in here. Very small detail but thought I should point out.

File: lesson03/assignment/index.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      "email": "mJordan@bulls.com",
+      "linkedInUrl": "mJordas.linkedinprofile.com",
+      "codeLanguages": [
+         "HTML", "CSS", "JavaScript", "Java", "Spring"
+      ]
+    }
+   ]
+
+// Card object code, has data fields and 1 method that returns a string of html to be inserted into page
+class Card {
```
Comment (tylervanbrocklin) [inline lesson03/assignment/index.js:57]: Excellent work with creating an object for your cards.

File: lesson03/assignment/index.js (hunk 1 above)
Comment (paulfleurdelys) [reply lesson03/assignment/index.js:1]: Thanks Tyler, I wasn't sure if changing the data.json to a data.js file and pulling it in via the script tag was incorrect, so I wanted to leave this option in as an alternative.  Thanks for the review!

**Comment by tylervanbrocklin** [comment]
Ahhh no worries at all, we probably should’ve clarified this! Nicely done on the assignment Paul.

Thank you,
Tyler Van Brocklin

**Pull request merged**